poem/src/server.rs

```diff
@@ -135,9 +135,8 @@ where
                         let notify = notify.clone();
                         let timeout_notify = timeout_notify.clone();
 
+                        alive_connections.fetch_add(1, Ordering::SeqCst);
```

Comment:
Btw, I see `SeqCst` is used here and it's strict. 
Probably it should be refactored to `Acquire/Release` 🤔

Reply:
Could you help me with this? 🙂

Reply:
Sure!
As separate PR or this one?

Reply:
thanks 🙂